docs/version0.39.md

```diff
@@ -27,17 +27,37 @@
 The following new features and notable changes since version 0.38.0 are included in this release:
 
 - [New binaries and changes to supported environments](#binaries-and-supported-environments)
+- [New JDK 20 features](#new-jdk-20-features)
 
 ## Features and changes
 
 ### Binaries and supported environments
 
 Eclipse OpenJ9&trade; release 0.39.0 supports OpenJDK 20.
 
+OpenJDK 20 with Eclipse OpenJ9 is *not* a long term support (LTS) release.
+
 RHEL 8.4 is out of support. RHEL 8.6 is the new minimum operating system level.
 
 To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md).
 
+### ![Start of content that applies to Java 20 plus](cr/java20plus.png) New JDK 20 features
+
+The following features are supported by OpenJ9:
+
+- [JEP 434](https://openjdk.java.net/jeps/434): Foreign Function & Memory API (Second Preview)
+- [JEP 436](https://openjdk.java.net/jeps/436): Virtual Threads (Second Preview)
+- [JEP 437](https://openjdk.java.net/jeps/437): Structured Concurrency (Second Incubator)
```

Comment:
Pls remove this one.

Reply:
@babsingh is Structured Concurrency supported now in the 0.39 branch?

Reply:
@pshipton - So I will retain this JEP 437 information?

Reply:
Yes, thanks.